pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -328,8 +328,15 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	// Initialize CPU manager
 	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManager) {
+		cpuManagerPolicyOptions := nodeConfig.ExperimentalCPUManagerPolicyOptions
+		if !utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManagerPolicyOptions) {
+			klog.InfoS("CPU Manager policy options feature gate not enabled, ignoring values", "option", cpuManagerPolicyOptions)
+			cpuManagerPolicyOptions = nil
+		}
```

Comment:
We probably want this to default to unset and only set if the feature gate is on, rather than the other way around?

Reply:
good point, let me fix it